### ISSUE TITLE
Web - Develop Social Compliance System - Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#73]
The SVG left-pointing arrow is missing a title, description, or some piece of ARIA as part of the "Homepage" link. Same applies to the bookmarks icon. There are five other images that I couldn’t see without having my tools in IE to find that are empty or missing ALT text.

There might be missing ALT text on other pages that I didn't have time to check so please those too.

Develop Social Compliance System - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/why-develop

**Comment by KyleLivingstonDOL** [comment]
Thanks John. Can you please specify what you mean by "five other images"?

ZTI, can you please address the "back" arrow that is attached to HOMEPAGE?

**Comment by jfc3-dol** [comment]
Should be for the four icons in the footer. Notice there is is a fainted dashed line around the other three in the footer,

Please see attached screenshot.

![comply chain - web - icons in footer](https://user-images.githubusercontent.com/10423487/61250465-c0b18a00-a725-11e9-9830-3f9690ac9b29.png)

**Comment by Abychalil** [comment]
alt text provided the Home Page Button..
Fixed

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for adding the necessary ALT text. Will look at it when the next version/build is sent, and I have time to test.

**Comment by jfc3-dol** [comment]
There is now ALT text on the left pointing arrow and other items, so I'm going to close the issue.
